.github/workflows/phpcs.yml

```diff
@@ -0,0 +1,45 @@
+name: Drupal coding standards
+
+on:
+  # Run on all pushes and on all pull requests.
+  # Prevent the build from running when there are only irrelevant changes.
+  push:
+    paths-ignore:
+      - '**.md'
+  pull_request:
+    paths-ignore:
+      - '**.md'
+  # Allow manually triggering the workflow.
+  workflow_dispatch:
+
+jobs:
+  Verify:
+    name: "Verify Drupal conding standards"
+
+    runs-on: "ubuntu-latest"
+
+    env:
+      CODE_SNIFFER_PATH: "modules/custom/ themes/custom/"
+      CODE_SNIFFER_IGNORE: "*md,*.css,*.txt,*.info,*interactive_ui/react_slider/js*,*pl-sk/*,*pattern-lab/*,*_twig-components/*"
+      CODE_SNIFFER_EXTENSIONS: "php,inc,install,module,yaml,twig,theme"
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Install PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '7.4'
```

Comment:
How about settings the version also in the `env` ?

Reply:
Php version moved to env